src/Components/test/testassets/ComponentsApp.Server/ComponentsApp.Server.csproj

```diff
@@ -10,6 +10,7 @@
 
   <ItemGroup>
     <Reference Include="Microsoft.AspNetCore.Components.Server" />
+    <Reference Include="Microsoft.AspNetCore.Mvc" />
```

Comment:
@rynowak I may have lost track of some of the discussions, but wasn't the idea that we'd be eliminating MVC dependencies from our Components test projects so that we could eliminate the long dependency chain (that goes right into IIS) from the Components solution? Does this change block that plan, or am I falsely conflating issues?

Reply:
I'm not sure, but this is temporary for preview3

Reply:
that is part of the plan and it does. But I know @javiercn plans for this to be temporary.